index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Indie+Flower&family=Paprika&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Spline+Sans:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="nav">
+      <a href="#" class="logo"><h1>My logo</h1></a>
+      <nav><i class="fas fa-bars menu_icon"></i></nav>
+    </header>
```

Comment:
- [ ] **Required:** I have observed that your design doesn't conform to that of the Figma template you've chosen. I have shown some of them below:

Your Design | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150539560-a4aa8426-193a-4430-972b-ea3e23e3223d.png) | ![image](https://user-images.githubusercontent.com/27709832/150540141-5c74d327-06f2-48e7-8cf6-737b41e6a899.png) 
![image](https://user-images.githubusercontent.com/27709832/150540033-d4e442ab-142d-48a4-a257-50adf488ca14.png) | ![image](https://user-images.githubusercontent.com/27709832/150540269-d07a9c14-55dd-4a71-80cf-a2f691e46365.png)